messages/SfgeEngine.js

```diff
@@ -1,4 +1,12 @@
 module.exports = {
-	"spinnerStart": "Analyzing with Salesforce Graph Engine. See %s for details.",
-	"pleaseWait": "Please wait"
+	"messages": {
+		"pleaseWait": "Please wait",
+		"spinnerStart": "Analyzing with Salesforce Graph Engine. See %s for details."
+	},
+	"errors": {
+		"failedWithoutProjectDir": `Salesforce Graph Engine cannot run without --projectdir/-p. You must rerun your command, and either use --projectdir/-p so Graph Engine can run, or use --engine/-e to exclude Graph Engine from execution.`
+	},
+	"warnings": {
+		"skippedWithoutProjectDir": `Salesforce Graph Engine cannot run without --projectdir/-p, and will be skipped due to your %s.%s value of %s in %s.`
```

Comment:
@teresa-allen-sfdc , new message. If the `missingOptionsBehavior` config property is set to `warn`, then a `scanner:run` execution that tries to run GraphEngine without `--projectdir/-p` will print this as a warning before skipping GraphEngine and continuing with other engines.